overview.html

```diff
@@ -0,0 +1,26 @@
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>FoodBlog</title>
+    <link rel="stylesheet" type="text/css" href="style.css"
+  </head>
+  <body>
+    <div class="header">
+      <h1 class="top-header">Recipes</h1>
+    </div>
+    <div class="category">
+      <div
+        class="category-block"
+        <img
+        src="./images/ricenoodle.jpg"
+        link="details.html"
+        width="360"
+        height="240"
+        alt="food"
+      >
+        PHO BO
+      </div>
```

Comment:
Two closing tags are missing:
```suggestion
      <div class="category-block">
        <img
          src="./images/ricenoodle.jpg"
          link="details.html"
          width="360"
          height="240"
          alt="food"
        />
        PHO BO
      </div>
```